docs/test_runner.md

```diff
@@ -0,0 +1,48 @@
+# ELECCIÓN TEST RUNNER
+
+Para hacer elección del Test-Runner que mejor se adapte a mi proyecto los voy a comparar y voy a elegir el que mejor se adapte a mi proyecto. Para ello me voy a ayudar de [snyk](https://snyk.io/advisor/).
+
+Los criterios que voy a seguir son:
+
+1. **Freshness:** es muy importante que test runner a elegir esté atualizado y que se actualice cada poco tiempo.
+2. **Buenas prácticas:** ver qué es lo que aconseja la mayoría de usuarios de node.js.
+3. **Velocidad:** es importante que el test se ejecute rápido.
+4. **Biblioteca de aserciones**: buscaré que el test runner tenga su propia biblioteca de aserciones para evitar la deuda técnica.
+
+
+## jest
+
+- En cuanto al **mantenimiento** (freshness), jest está en contante actualización para mejorar.
+- Este test es **rápido** en su ejecución ya que se puede ejecutar en paralelo.
+- En cuanto a la valoración general de [**snyk**](https://snyk.io/advisor/npm-package/jest), tiene una valoración alta por parte de los usuarios. Consta de comunidad activa.
+- Es el más usado actualmente por los usuarios. Ya que las pruebas están aisladas unas de otras, es rápido y ofrece diferentes tipos de excepciones para las pruebas.
```

Comment:
¿Esto corresponde a "lo que aconsejan la mayoría de los usuarios"? ¿Dónde lo has medido objetivamente y como?

Reply:
He usado la información de estos enlaces: [Top JavaScript Testing Frameworks](https://www.browserstack.com/guide/top-javascript-testing-frameworks) y [State of JS: Testing](https://2022.stateofjs.com/en-US/libraries/testing/)